### DISCUSSION TITLE
Remove branding in prod [Feb 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5086] [Discord Threads]
> 🙋‍♂️ **shikishikichangchang**

**Comment by o1lab** [comment]
> 🙋‍♂️ **shikishikichangchang**  
#### Is there way where I can remove all branding in production?  
#  
> 🙋‍♂️ **shikishikichangchang**  
#### Eg remove this  
#  
> 🙋‍♂️ **shikishikichangchang**  
####   
![image0](https://cdn.discordapp.com/attachments/1061120255080018031/1061120438748586014/image.png)  
#  
> 🙋‍♂️ **shikishikichangchang**  
####   
![image0](https://cdn.discordapp.com/attachments/1061120255080018031/1061120503009525770/image.png)  
#  
> 🙋‍♂️ **shikishikichangchang**  
#### And this, since users wont need the api docs etc  
#  
> 🙋‍♂️ **shikishikichangchang**  
####   
![image0](https://cdn.discordapp.com/attachments/1061120255080018031/1061120591085711370/image.png)  
#  
> 💁 **o1lab**  
#### not at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1061120255080018031)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by x0rb** [comment]
I don't mind a pretty nocodb branding. As it is now, it's obnoxious. 

There's even a "Signup for free" button top right that directs to NocoDB.com's signup page. 

I like where NocoDB is going, but I can't recommend it for startups looking to outgrow the free managed or self-hosted plan with how obnoxious their branding is. When branding is brought up, one of the dev suggests forking it knowing full well none will. Pretty insulting. A matter of time before future release end up becoming more restrictive under a dual license. 

Self-hosted is awesome for the hobbyist though.

![nocodb-obnoxious-branding](https://github.com/user-attachments/assets/d8c8a3cb-18e3-4372-adcf-4cc27fe51def)

**Comment by cmdaltctr** [comment]
I don't mind the Powered by Nocodb tbh. But the banner, logo (cannot change unless enterprise) - that's really, as the comment earlier said, obnoxious. These types of features should be available as they separate Nocodb and others like Seatable, Airtable, etc. By hide branding usually means you can't remove "powered by", which I can accept truly.

Secondly, the enterprise push is also too much. Sharing a link to a custom domain requires enterprise as well..?? Nocodb was a golden alternative to Baserow, which I looked forward to (as Baserow is a freemium model and, quite frankly, useless as a free product (for example, I could do the same and more with Grist Cloud). You get the idea.

I get that Nocodb needs to make some money to survive, but you could try charging for some cool plugins or integration, especially for self-host. The latter would mean making your own, which most people don't want to do if they can avoid it. 
Follow the n8n model, no locked-in or paywall, just pure performance, and they are doing really, really well.

**Comment by ReessKennedy** [comment]
IDEA: If you are just annoyed by branding you can pretty easily run it behind a proxy or Cloudflare and then just run some JS on top of Nocodb that replaces the logo with anything you want (or your company's logo) and can replace any mention of Nocodb in the HTML as well. Just use a worker in Cloudflare and ask ChatGPT how to write the Javascript. Cloudflare allows you to deploy this globally to all users of your install but if it's only you then you can just insert this only on your machine using a JS injection chrome extension.
